$OpenBSD: patch-src_lib_synergy_CProtocolUtil_h,v 1.2 2013/05/12 16:09:04 bcallah Exp $

Move private function to public so loongson can use it.

--- src/lib/synergy/CProtocolUtil.h.orig	Mon Sep  3 22:09:56 2012
+++ src/lib/synergy/CProtocolUtil.h	Thu Apr 18 10:50:31 2013
@@ -72,6 +72,7 @@ class CProtocolUtil { (public)
 	static bool			readf(synergy::IStream*,
 							const char* fmt, ...);
 
+	static void			writef(void*, const char* fmt, va_list);
 private:
 	static void			vwritef(synergy::IStream*,
 							const char* fmt, UInt32 size, va_list);
@@ -79,7 +80,6 @@ class CProtocolUtil { (public)
 							const char* fmt, va_list);
 
 	static UInt32		getLength(const char* fmt, va_list);
-	static void			writef(void*, const char* fmt, va_list);
 	static UInt32		eatLength(const char** fmt);
 	static void			read(synergy::IStream*, void*, UInt32);
 };
